### PR TITLE
Added clarification on version to download for Umbraco Forms theme files.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added clarification on version to download for Umbraco Forms theme files to make the confusion discussed under https://github.com/umbraco/Umbraco.Forms.Issues/issues/1057 less likely.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [X] Other

## Product & version (if relevant)

Umbraco Forms 10+

## Deadline (if relevant)

Can be reviewed and go live at any time.

### CODE REVIEW

**Comment by AndyButland** [comment]
Remaining linting error looks to be a false report of a sentence longer than 25 words.

File: 11/umbraco-forms/developer/themes.md [hunk 1]
```diff
@@ -20,10 +20,16 @@ Umbraco 10+ distributes these files as part of a Razor class library, so you won
...
+* [Forms 11 Default Theme](https://umbra.co/umbraco-forms-default-theme-11) (for 11.1.0 and above)
 
-The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available. So if you are running Umbraco 11.1, and no file for that version is available, use the one for Umbraco 11.0.
+If you are using a lower minor version of Forms than those listed, you should download an older version of the default theme.
+
+The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.
+
+You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.
+
+For example, you might be using Umbraco Forms 10.4. No file for that version is available so you should use the one for 10.3.
```
Comment (alina-tincas) [inline 11/umbraco-forms/developer/themes.md:32]:
```suggestion
For example, when using Umbraco Forms 10.4, and no file for that version is available use version 10.3 instead.
```

File: 12/umbraco-forms/developer/themes.md [hunk 2]
```diff
@@ -20,11 +20,17 @@ Umbraco 10+ distributes these files as part of a Razor class library, so you won
...
+* [Forms 12 Default Theme](https://umbra.co/umbraco-forms-default-theme-12) (for 12.1.0.rc1 and above)
 
-The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available. So if you are running Umbraco 11.1, and no file for that version is available, use the one for Umbraco 11.0.
+If you are using a lower minor version of Forms than those listed, you should download an older version of the default theme.
+
+The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.
+
+You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.
+
+For example, you might be using Umbraco Forms 10.4. No file for that version is available so you should use the one for 10.3.
```
Comment (alina-tincas) [inline 12/umbraco-forms/developer/themes.md:33]:
```suggestion
For example, when using Umbraco Forms 10.4, and no file for that version is available use version 10.3 instead.
```

File: 10/umbraco-forms/developer/themes.md [hunk 3]
```diff
@@ -5,23 +5,39 @@ description: Documentation on how to apply custom themes to Umbraco Forms
...
+
+* [Forms 10 Default Theme](https://umbra.co/umbraco-forms-default-theme) (for 10.5.0.rc1 and above)
+
+If you are using a lower minor version of Forms than those listed, you should download an older version of the default theme.
+
+The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.
+
+You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.
+
+For example, you might be using Umbraco Forms 10.4. No file for that version is available so you should use the one for 10.3.
```
Comment (alina-tincas) [inline 10/umbraco-forms/developer/themes.md:32]:
```suggestion
For example, when using Umbraco Forms 10.4, and no file for that version is available use version 10.3 instead.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @AndyButland thank you for the PR! 🙌 

I have a few suggestions so I was wondering if you can look through them and let me know how they sound? 😊 

Some of the suggestions might be outdated as I have started the review before you made your latest changes 🙈

File: 12/umbraco-forms/developer/themes.md (hunk 2 above)
Comment (AndyButland) [reply 12/umbraco-forms/developer/themes.md:33]: I think this one is OK as is (I had already changed it due to the "sentence too long" warning).

File: 11/umbraco-forms/developer/themes.md (hunk 1 above)
Comment (AndyButland) [reply 11/umbraco-forms/developer/themes.md:32]: I think this one is OK as is (I had already changed it due to the "sentence too long" warning).

File: 10/umbraco-forms/developer/themes.md (hunk 3 above)
Comment (AndyButland) [reply 10/umbraco-forms/developer/themes.md:32]: I think this one is OK as is (I had already changed it due to the "sentence too long" warning).

File: 10/umbraco-forms/developer/themes.md [hunk 4]
```diff
@@ -5,23 +5,39 @@ description: Documentation on how to apply custom themes to Umbraco Forms
...
 
-For Umbraco 9 and previous, it's straightforward to simply copy the files you need from the default theme folder. Umbraco 10 distributes these files as part of a Razor class library, so you won't find them on disk. The easiest way to obtain copies of the files included in the default theme is to [download this zip file](https://our.umbraco.com/FileDownload?id=23911) and extract the ones you need.
+### Obtaining the Default Theme Files
+
+For Umbraco 9 and previous, it's straightforward to copy the files you need from the default theme folder. We highly recommend that you never customize any files found in the `default` themes folder. There is a risk that any customizations to these files will be lost with any future upgrades you do to Umbraco Forms.
+
+Umbraco 10+ distributes these files as part of a Razor class library, so you won't find them on disk. Instead you should download the appropriate zip file for your Forms version and extract the ones you need.
+
+You can obtain the latest versions of the default theme for each Forms major version from the following links:
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 10/umbraco-forms/developer/themes.md:22]: Given there's only one link for this version, probably best as:

```suggestion
You can obtain the latest version of the default theme Forms 10 from the following link:
```

**Comment by AndyButland** [comment]
Thanks @alina-tincas - have accepted or responded on the suggestions.

File: 11/umbraco-forms/developer/themes.md [hunk 5]
```diff
@@ -20,10 +20,16 @@ Umbraco 10+ distributes these files as part of a Razor class library, so you won
 
 You can obtain the latest versions of the default theme for each Forms major version from the following links:
 
-* [Forms 10 Default Theme](https://umbra.co/umbraco-forms-default-theme)
-* [Forms 11 Default Theme](https://umbra.co/umbraco-forms-default-theme-11)
+* [Forms 10 Default Theme](https://umbra.co/umbraco-forms-default-theme) (for 10.5.0.rc1 and above)
+* [Forms 11 Default Theme](https://umbra.co/umbraco-forms-default-theme-11) (for 11.1.0 and above)
 
-The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available. So if you are running Umbraco 11.1, and no file for that version is available, use the one for Umbraco 11.0.
+If you are using a version of Forms of a lower minor version than those listed, you should download an older version of the default theme.
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 11/umbraco-forms/developer/themes.md]: Resolving as outdated.

File: 10/umbraco-forms/developer/themes.md [hunk 6]
```diff
@@ -5,23 +5,39 @@ description: Documentation on how to apply custom themes to Umbraco Forms
...
+
+* [Forms 10 Default Theme](https://umbra.co/umbraco-forms-default-theme) (for 10.5.0.rc1 and above)
+
+If you are using a version of Forms of a lower minor version than those listed, you should download an older version of the default theme.
+
+The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.
+
+You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.
+
+For example, if you were using Umbraco Forms 10.4, and no file for that version is available, you would use the one for 10.3.
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 10/umbraco-forms/developer/themes.md]: Resolving as outdated.

File: 11/umbraco-forms/developer/themes.md [hunk 7]
```diff
@@ -20,10 +20,16 @@ Umbraco 10+ distributes these files as part of a Razor class library, so you won
...
+
+The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.
+
+You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.
+
+For example, if you were using Umbraco Forms 10.4, and no file for that version is available, you would use the one for 10.3.
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 11/umbraco-forms/developer/themes.md]: Resolving as outdated.

File: 12/umbraco-forms/developer/themes.md [hunk 8]
```diff
@@ -20,11 +20,17 @@ Umbraco 10+ distributes these files as part of a Razor class library, so you won
...
+* [Forms 12 Default Theme](https://umbra.co/umbraco-forms-default-theme-12) (for 12.1.0.rc1 and above)
 
-The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available. So if you are running Umbraco 11.1, and no file for that version is available, use the one for Umbraco 11.0.
+If you are using a version of Forms of a lower minor version than those listed, you should download an older version of the default theme.
+
+The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.
+
+You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.
+
+For example, if you were using Umbraco Forms 10.4, and no file for that version is available, you would use the one for 10.3.
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 12/umbraco-forms/developer/themes.md]: Resolving as outdated.

File: 10/umbraco-forms/developer/themes.md (hunk 3 above)
Comment (alina-tincas) [reply 10/umbraco-forms/developer/themes.md:32]: Thank you for confirming! I believe in this case it is best to have the 2 this content together so it is more understandable that it is related to the example with 10.4. 

So I will just merge this one "For example, when using Umbraco Forms 10.4, and no file for that version is available use version 10.3 instead." 👍

**Review by alina-tincas — APPROVED** [review]

**Pull request merged**